### ISSUE TITLE
Sorry, your request failed. Please try again. Request id: 1eda63f0-a865-4fbd-98e1-c20f3cff69e4

### ISSUE THREAD

**Issue by Raghuna** [microsoft/vscode-copilot-release#3532]
- Copilot Chat Extension Version:
- VS Code Version:
- OS Version:
- Logs:

Steps to Reproduce:

1.
2.

Please resolve this issue

**Comment by sandy081** [comment]
Can you please let us know if it worked after retrying?

**Comment by Raghuna** [comment]
yes but today i am repeatedly getting this issue, Sorry, your request failed. Please try again. Request id: 5d23eaa2-72c4-46ab-a2b1-43872fffc0f6
